internal/beater/config/rum.go

```diff
@@ -76,7 +76,7 @@ func (c *RumConfig) setup(log *logp.Logger, outputESCfg *config.C) error {
 
 	// Unpack the output elasticsearch config first
 	if err := outputESCfg.Unpack(c.SourceMapping.ESConfig); err != nil {
-		return errors.Wrap(err, "unpacking Elasticsearch output config into Sourcemap config")
+		return fmt.Errorf("unpacking Elasticsearch output config into Sourcemap config: %w", err)
```

Comment:
Should it be like this to keep consistent with other errors?

```suggestion
		return fmt.Errorf("unpacking elasticsearch output config into sourcemap config: %w", err)
```